docs/advanced-features/custom-document.md

```diff
@@ -35,6 +33,8 @@ class MyDocument extends Document {
 export default MyDocument
 ```
 
+> The code above is the default `Document` added by Next.js. Feel free to remove it if you don't need to change it.
```

Comment:
I think we can be more specific:

```suggestion
> The code above is the default `Document` added by Next.js. Feel free to remove the `getInitialProps` or `render` function from `MyDocument` if you don't need to change them.
```

Reply:
What I mean by this change is that you can remove the entire file, sure it will work without any of those methods too but it doesn't matter in that case.